modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/What.java

```diff
@@ -19,18 +19,26 @@
  */
 package com.spotify.apollo.metrics.semantic;
 
-import com.codahale.metrics.Timer;
-import com.spotify.apollo.metrics.TimerContext;
+/**
+ * Enumerates the metrics tracked, keyed by the 'what' tag.
+ */
+public enum What {
```

Comment:
I'm thinking of the `What`:s as making up the API to users - it's fairly convenient to have a well-defined list of metrics that you can choose to generate or not. And IntelliJ reports that it could be package private, but that's a bug; this branch doesn't compile if you make that change, since `What` is used in a lambda in the MetricsModule, currently. So I think this should be public.